cmd/kava/opendb/opendb_rocksdb_test.go

```diff
@@ -337,7 +337,7 @@ func TestNewRocksDBWithOptions(t *testing.T) {
 	cfOpts := newDefaultOptions()
 	cfOpts.SetWriteBufferSize(999_999)
 
-	db, err := newRocksDBWithOptions(name, dir, dbOpts, cfOpts)
+	db, err := newRocksDBWithOptions(name, dir, dbOpts, cfOpts, true, defaultReportMetricsIntervalSecs)
```

Comment:
Nit: do we need to extend this method signature or can these opts be added onto the dbOpts struct?

Reply:
They can't be added to `dbOpts`, but they are coming from `appOpts`, so I can change signature from:
```go
func newRocksDBWithOptions(..., enableMetrics bool, reportMetricsIntervalSecs int64)
```
to
```go
func newRocksDBWithOptions(..., appOpts types.AppOptions)
```

but I wasn't sure about this because it's a bit harder to `unit-test` such function, because you have to use `MockAppOptions` instead of just passing args so I decided to parse `appOpts` in the highest-level function

but it's debatable decision